api/search/queryprocessor.py

```diff
@@ -159,7 +159,7 @@ def _get_targets(self, path):
                     c: [copy.deepcopy(target)] for c in self.containers
                 }
             else:
-                return {path_parts[-min_length-1]: [target]}
+                return {path_parts[-2]: [target]}
```

Comment:
Ah no worries I just wasn't sure if what I filled in would have the intended functionality. Is `-2` (grabbing the container in the path above `file` or `notes`) a good enough replacement for now?